src/batchUploader.ts

```diff
@@ -108,11 +109,11 @@ export class BatchUploader {
             _Store: { deviceId },
         } = this.mpInstance;
 
-        const offlineStorageFeatureFlagValue = getFeatureFlag(
+        const offlineStorageFeatureFlagValue: string = getFeatureFlag(
```

Comment:
Let's add this to sustained engineering.

```suggestion
        // https://go.mparticle.com/work/SQDSDKS-6317
        const offlineStorageFeatureFlagValue: string = getFeatureFlag(
```